src/components/WebLogView/ContentPreview.tsx

```diff
@@ -22,13 +25,20 @@ export function ContentPreview({
   contentType,
   searchIndex,
   searchString,
+  originalContent,
 }: ContentPreviewProps) {
   const [selectedTab, setSelectedTab] = useState('content')
 
   useEffect(() => {
     setSelectedTab('content')
   }, [format])
 
+  useEffectOnce(() => {
```

Comment:
Some thoughts about this one:
1. What do you think about displaying the diff in the content tab? I haven't thought this through and it might not be a good idea since "raw" and "preview" will not have this info and it might be confusing.
2. If the request data is unaffected, shouldn't we hide the diff option? 
3. Regarding `useEffectOnce` specifically, is it here just to set a different default value? If so, it's worth moving this logic to line 30 IMO:
```ts
const [selectedTab, setSelectedTab] = useState(originalContent && originalContent !== content && !searchString ? 'diff' : 'content')
```

Reply:
I think it's till worth keeping content and diff tabs separate, just to have an option to view content without diff highlights. Other comments addressed 👍